ncm-spma/src/main/pan/components/spma/config.pan

```diff
@@ -5,10 +5,10 @@
 
 unique template components/${project.artifactId}/config;
 
-variable CONFIG_MODULES_CONFIG_SUFFIX ?= 'rpm';
+variable SPMA_BACKEND ?= 'yum';
```

Comment:
As for the variable name change, it's fine with me. I checked in the template library and found no usage of it outside `ncm-spma`.